setup.cfg

```diff
@@ -19,7 +19,6 @@ keywords =
 classifiers =
     Programming Language :: Python :: 3 :: Only
     Programming Language :: Python :: 3.8
-    License :: OSI Approved :: GNU General Public License v3 (GPL-3.0)
```

Comment:
The PYPI release from Github actions was failing because of this line, so it has been removed. We already have license information in the metadata section of setup.cfg.